site/_posts/2018-02-14-the-pathname-to-discovering-a-memory-leak-in-ruby.html.md

```diff
@@ -0,0 +1,429 @@
+---
+title: The Path[name] to discovering a memory leak in Ruby
+author: NickLaMuro
+date: 2018-02-14
+tags: ruby performance
```

Comment:
I suggest using the existing `developers` tag in place of `performance`. We don't have many development-related posts for now to break them down so finely. If this changes in the future we can add the tags back :)